### PR TITLE
Updated contributing.md

### PR DESCRIPTION
**Pull request by Tutu6790**
Added a few more contribution guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,7 +21,13 @@ In this project, you will work on image classification tasks. We have provided a
 
 4. Work on your image classification tasks in your notebook.
 
-5. When you are ready, make a pull request to the main repository.
+5. The minimum contribution a user can make is data processing.
```
Comment (Sammybams) [inline CONTRIBUTING.md:24]:
```suggestion
5. The minimum contribution you can make is on preprocessing of the data(images).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,7 +21,13 @@ In this project, you will work on image classification tasks. We have provided a
...
+
+6. For contributions related to modeling, the model's performance must achieve a minimun of 60% accuracy to be considered valid.
+
+7. The team will conduct test on model using a specific dataset to verify its performance score.
```
Comment (Sammybams) [inline CONTRIBUTING.md:28]:
```suggestion
7. The team will perform an evaluation of the model using a separate test dataset to verify its performance score.
```

**Review by Sammybams — APPROVED** [review]
Thank you very much for these updates. I have also made some changes to the wording in lines 24 and 28.

**Comment by Tutu6790** [comment]
Thank you. do i need to add those changes and make another pull requests?

**Comment by Sammybams** [comment]
Yes, you would need to make those changes.

**Review by Tutu6790 — COMMENTED** [review]
changes updated. please check to confirm

File: CONTRIBUTING.md (hunk 2 above)
Comment (Sammybams) [inline CONTRIBUTING.md:28]: lkml

**Pull request merged**